agreements/activity-types.md

```diff
@@ -0,0 +1,19 @@
+# Activity Types Agreement
+
+We have assessed that there are four types of activities that members carry out within Root Systems:
+
+## External
+
+This is contracting work with the outside world - usually software development, but could include any other types of consulting work.
+
+## Internal Investment
+
+This is either internal work towards development of a Root Systems product, or personal development / upskilling of a member (i.e. conferences, training).
+
+## Internal Administration
+
+This is time spent towards the administration of the business - typically work done as part of either a [coordinator](../roles/coordinator) or [director](../roles/director) role.
```

Comment:
Differentiating between the two comes from a conversation (i think mainly) between dan and I, basically the intention is to recognise that working on administration of the company and working on internal products are different activities, particularly with regard to how much members want to do them! I think there is value in recognising them separately, even if we don't pay them differently, but open to debate on this.

I would say invoicing clients is an example yes. I agree delegation of these tasks is desirable, i could make that more clear that they can be delegated.